internal/impl/devbox.go

```diff
@@ -171,11 +173,6 @@ func (d *Devbox) Shell(ctx context.Context) error {
 		return err
 	}
 
-	pluginHooks, err := plugin.InitHooks(d.mergedPackages(), d.projectDir)
-	if err != nil {
-		return err
-	}
-
 	env, err := d.nixEnv(ctx)
```

Comment:
somewhere around here, we should ensure the script files are written. see the `run` impl

you can test if they're being written by deleting `.devbox/gen/scripts` and then running `devbox shell`

Reply:
on line 164: d.ensurePackagesAreInstalled
-> d.generateShellFiles
-> d.writeScriptsToFiles